RussianFilter/sections/adservers.txt

```diff
@@ -2,6 +2,7 @@
 ! Содержит доменные имена, использующиеся рекламными сетями.
 !
 ||metrika.traff.space^$third-party
+||trfmxt.com^$third-party
```

Comment:
Domain not requested when I check the site.

Reply:
<details><summary>Screenshot:</summary>

![lf](https://user-images.githubusercontent.com/33488229/64948318-78c2f680-d87f-11e9-9fcd-8af9d56fd0d9.png)
</details><br/>